stanford_fwb.features.inc

```diff
@@ -19,3 +19,27 @@ function stanford_fwb_ctools_plugin_api($module = NULL, $api = NULL) {
 function stanford_fwb_views_api($module = NULL, $api = NULL) {
   return array("api" => "3.0");
 }
+
+/**
+ * Implements hook_image_default_styles().
+ */
+function stanford_fwb_image_default_styles() {
```

Comment:
oh boy. So this module was dependant on an image style on a non-standard branch? yikes.